test/infrastructure/container/docker.go

```diff
@@ -508,8 +509,8 @@ func (d *dockerRuntime) RunContainer(ctx context.Context, runConfig *RunContaine
 		err := errors.Wrapf(err, "error starting container %q", runConfig.Name)
 		// Delete the container and retry later on. This helps getting around the race
 		// condition where of hitting "port is already allocated" issues.
-		if innerErr := d.dockerClient.ContainerRemove(ctx, resp.ID, types.ContainerRemoveOptions{Force: true, RemoveVolumes: true}); innerErr != nil {
-			return errors.Wrapf(innerErr, "error removing container after failed start: %s", err)
+		if reterr := d.dockerClient.ContainerRemove(ctx, resp.ID, types.ContainerRemoveOptions{Force: true, RemoveVolumes: true}); reterr != nil {
+			return kerrors.NewAggregate([]error{reterr, errors.Wrapf(err, "error deleting container")})
```

Comment:
I think this mixes up the errors, sorry :D 

The `err` from above should not be further wrapped, right?

(The difference compared to the example I provided is that you're reterr is the other one

Reply:
So this would work
```go
return kerrors.NewAggregate([]error{err, errors.Wrapf(reterr, "error deleting container")})
```